src/App.vue

```diff
@@ -1,12 +1,41 @@
 <template>
   <div id="app">
-    <h1>hello world</h1>
+    <tx-preloader v-if="preload" :progress="progress" />
+
+    <main v-else>
+      <h1>hello world</h1>
+    </main>
   </div>
 </template>
 
 <script>
-export default {}
+import Preloader from "./assets/components/atoms/Preloader.vue";
+
+export default {
+  components: {
+    "tx-preloader": Preloader,
+  },
+  data() {
+    return {
+      preload: true,
```

Comment:
please use Boolean convention. Reference: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

Reply:
am going to change it to `isLoad`, does that sound good?

Reply:
`isLoading` better. present tense = (is/am/are) + (verb+ing)

Reply:
okay, thank you for the explanation